http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1778,6 +2079,108 @@ components:
         patchInProgress:
           type: boolean
           description: Whether there is a patch in progress on the runtime. Is used to indicate updates that require status transitions.
+    ListPersistentDiskResponse:
+      description: ""
+      required:
+        - id
+        - googleProject
+        - zone
+        - name
+        - googleId
+        - samResourceId
```

Comment:
I thought you were going to remove this `samResourceId`?

Reply:
It turns out we use it ourselves to check whether or not the disk is visible for the user before `list`ing, so I think it makes sense to keep in there